frontend/model/contracts/group.js

```diff
@@ -1361,10 +1369,16 @@ sbp('chelonia/defineContract', {
       }
     },
     'gi.contracts/group/changeChatRoomDescription': {
-      validate: actionRequireActiveMember(objectOf({
-        chatRoomID: string,
-        description: string
-      })),
+      validate: (data, props) => {
+        actionRequireActiveMember(objectOf({
+          chatRoomID: string,
+          description: string
+        }))(data, props)
+
+        if (data?.description.length > CHATROOM_DESCRIPTION_LIMITS_IN_CHARS) {
```

Comment:
This `data?` is unnecessary because `data` shouldn't be `null` (the previous check will throw if it is null).